app/assets/javascripts/hidden_until_select.js.coffee

```diff
@@ -0,0 +1,14 @@
+class HiddenUntilSelect
+  constructor: ($root, @$dependent) ->
+    @$controller = $root.find("##{ @$dependent.attr('data-hide-until-select') }")
+    @$controller.change => @checkHide()
+    @checkHide()
+
+  checkHide: ->
+    @$dependent.toggle(@$controller.val())
+
+$ ->
+  # @todo - better scope
```

Comment:
my view on todos: if it's important but not in scope, it should be in Tracker. If it's important but in scope: should be done now. If it's not important: don't do it!